internal/server/ptypes/project.go

```diff
@@ -81,6 +81,20 @@ func ValidateUpsertProjectRequest(v *pb.UpsertProjectRequest) error {
 	))
 }
 
+// ValidateGetProjectRequest
+func ValidateGetProjectRequest(v *pb.GetProjectRequest) error {
+	return validationext.Error(validation.ValidateStruct(v,
+		validation.Field(&v.Project, validation.Required),
+	))
+}
+
+// ValidateGetProjectRequest
```

Comment:
[sand] This comment doesn’t match the function below it. But also: these comments don’t add much, could we drop them altogether?

Reply:
Oops, yeah probably from all the copy paste to get the bare functions around before adding the content :D I can just drop the comment.

Reply:
If I were to do this again I'd probably delete them but now that they are there I'll probably leave them be 😅